src/index.html

```diff
@@ -14,6 +14,5 @@
         <div class="pop-throbber__image"></div>
       </div>
     </div>
-    <script src="application.js"></script>
   </body>
 </html>
```

Comment:
Note that I moved `src/static/index.html` to `src/index.html` since it's now used as a template for html-webpack-plugin.

Reply:
OK—I would say let’s maybe move it to `src/html/index.html` (mirroring `src/css/application.css`)? Realistically all the JS should be in `src/js` but that’s a reorganization for another day…